src/datetime/newInstance.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Returns a Date from its parts
+ * @param {number} year
+ * @param {number} month
+ * @param {number} day
+ * @param {number} hour
+ * @param {number} minutes
+ * @param {number} seconds
+ * @return Date
+ */
+
+import { DateTime } from "luxon";
```

Comment:
Is DateTime being used?

Reply:
Actually, it isn't. Fixed at [ae9b1b1](https://github.com/genexuslabs/web-standard-functions/pull/77/commits/ae9b1b1960b85aea7790bbcdeb8ee987be139f28)